packages/core/src/builder.class.ts

```diff
@@ -2033,6 +2033,10 @@ export class Builder {
     if (!this.apiKey) {
       throw new Error('Builder needs to be initialized with an API key!');
     }
-    return this.queueGetContent(modelName);
+    return this.queueGetContent(modelName, options);
+  }
+
+  getAll(modelName: string, options: GetContentOptions = {}) {
```

Comment:
Does the user still needs to add a `.promise()` to this call to be used with `await`?

Reply:
Also how about pagination?

Reply:
ah that's a good idea to .promise() by default for this

for pagination this supports the `offset` and `limit` params

Reply:
Since this is public API, can I suggest to add return types to this method?

Reply:
donzeo